doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+# Practical Overview of Explainability
+
+While the applications of machine learning are impressive many models provide predictions that are hard to interpret or reason about. This limits their use in many cases as often we want to know why and not just what a model made a prediction. Alarming predictions are rarely taken at face value and typically warrant further analysis. Indeed, in some
+cases explaining the choices that a model makes could even become a potential [legal requirement](https://arxiv.org/pdf/1711.00399.pdf). The following is a non-rigorous and practical overview of explainability and the methods alibi provide.
+
+**Explainability provides us with algorithms that give insights into trained models predictions.** It allows
+us to answer questions such as:
+- How does a prediction change dependent on feature inputs?
+- What features are Important for a given prediction to hold?
```

Comment:
Important --> important

In this section I would also use bold to emphasize key points, e.g. **important**, **not important**.

Reply:
Combine the "important" and "not important" features bullet points into 1.